bundles/org.openhab.binding.enigma2/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="enigma2"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<thing-type id="device">
+		<label>Enigma2</label>
+		<description>The Thing represents an Enigma2 device</description>
+		<channels>
+			<channel id="power" typeId="powerType" />
+			<channel id="mute" typeId="muteType" />
+			<channel id="volume" typeId="volumeType" />
+			<channel id="channel" typeId="channelType" />
+			<channel id="title" typeId="titleType" />
+			<channel id="description" typeId="descriptionType" />
+			<channel id="mediaPlayer" typeId="mediaPlayerType" />
+			<channel id="mediaStop" typeId="mediaStopType" />
+		</channels>
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<context>network-address</context>
+				<label>Host Address</label>
+				<description>Hostname or IP address of the Enigma2 device.</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" required="true" min="1" max="3600">
```

Comment:
Please also provide a default here.

Reply:
Fixed

Reply:
If there is a default why it is still required?